nucliadb_vectors2/src/vectors/data_point/ops_hnsw.rs

```diff
@@ -110,29 +111,29 @@ impl<'a, DR: DataRetriever> HnswOps<'a, DR> {
     }
     fn closest_up_node<L: Layer>(
         &'a self,
-        solution: &mut HashSet<Address>,
-        filters: &[&[u8]],
-        layer: L,
         x: Address,
-        y: Address,
+        query: Address,
+        layer: L,
+        filters: &[&[u8]],
+        sol_addresses: &HashSet<Address>,
```

Comment:
I would rather prefer `solution_addresses` (`sol` can stand for an acronyme)

Reply:
Will do

Reply:
fixed by [363880a](https://github.com/nuclia/nucliadb/pull/353/commits/363880a41a5053d81c5f3b42f5abbc68d8f4bd92)